modules/oe_content_project/config/install/field.field.node.oe_project.oe_documents.yml

```diff
@@ -0,0 +1,32 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_documents
+    - media.type.document
+    - node.type.oe_project
+  module:
+    - composite_reference
```

Comment:
This feels like it should have a dependency on oe_content_documents_field. Was this file generated through commands?

Reply:
yes, it was exported, this https://github.com/openeuropa/oe_content/blob/74b5f8dd7cde1f56fc32963525e565294101492d/modules/oe_content_project/config/install/field.field.node.oe_project.oe_documents.yml#L5 is how drupal exported the dependency, and in the info.yml we have the ref to the module

Reply:
It's normal since the module adds a third-party setting to every field config.